src/plone/staticresources/static/plone-base.js

```diff
@@ -0,0 +1,81 @@
+// Copyright (C) 2010 Plone Foundation
+//
+// This program is free software; you can redistribute it and/or modify it
+// under the terms of the GNU General Public License as published by the Free
+// Software Foundation; either version 2 of the License.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+// FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+// more details.
+//
+// You should have received a copy of the GNU General Public License along with
+// this program; if not, write to the Free Software Foundation, Inc., 51
+// Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+//
+
+if (window.jQuery) {
+  define('jquery', [], function () {
+    'use strict';
+    return window.jQuery;
+  });
```

Comment:
I'm not sure what the block above does (a comment would be nice, is it turning global jquery into a requirejs module?).

Do we need it both here and in plone.js?

Reply:
The block does exactly that.
Removed the doubled definition from plone.js locally and still seems to work